x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/RBACEngine.java

```diff
@@ -212,6 +212,7 @@ private static boolean shouldAuthorizeIndexActionNameOnly(String action, Transpo
             case "indices:data/write/reindex":
             case "indices:data/read/sql":
             case "indices:data/read/sql/translate":
+            case "indices:data/read/eql":
```

Comment:
was not sure if there is a better approach instead of this ^

Reply:
I am not sure we need this here. I don't see a scenario when we are going to resolve the index name based on the content of the request itself.

Reply:
Removing this change, replacing with more suitable solution after discussion with Igor.